rmgpy/data/solvationTest.py

```diff
@@ -193,6 +193,22 @@ def test_radical_and_lone_pair_generation(self):
         solute_data = self.database.get_solute_data_from_groups(species)
         self.assertIsNotNone(solute_data)
 
+    def test_radical_solute_group(self):
+        """Test that the existing radical group is found for the radical species when using group additivity"""
+        # First check whether the radical group is found for the radical species
+        rad_species = Species(molecule=[Molecule(smiles='[OH]')])
```

Comment:
You can simplify this species creation to `Species(smiles='[OH]')`.

Reply:
I applied this change!